### DISCUSSION TITLE
rose-pine theme isn't the same as the example (Lazyvim)

### DISCUSSION THREAD

**Discussion by Rujipas-Thongpao** [LazyVim/LazyVim#3404] [Q&A]
Im kinda new to the nvim and lazyvim and I want to use [rose-pine theme](https://github.com/rose-pine/neovim?tab=readme-ov-file)

# my config
![image](https://github.com/LazyVim/LazyVim/assets/56151961/54615164-d671-4b2d-9f26-e558eef3f06a)

# My nvim
![image](https://github.com/LazyVim/LazyVim/assets/56151961/3d7d2c8b-cb59-415d-a5a1-8a948c14d996)

# their example
![image](https://github.com/LazyVim/LazyVim/assets/56151961/6a069eb5-9f28-4559-ac47-4f2a2f39457e)

# Solution
it's simply because I use window terminal, which doesn't use the color properly.
to fix this I change it to Wezterm :)

**Comment by folke** [comment]
This is somthing you should ask in the repo of the theme.

Reasons why yours might be different: LSP semantic tokens, have/have not installed treesitter

**Comment by Rujipas-Thongpao** [comment]
I got it!
it's because I use window terminal which doesn't use color properly
TYSM!! 👍
